core/src/main/resources/hudson/views/Messages_zh_CN.properties

```diff
@@ -0,0 +1,32 @@
+# The MIT License
+#
+# Copyright (c) 2013, Chunghwa Telecom Co., Ltd., Pei-Tang Huang
```

Comment:
I am not sure the "year" is correct here

Reply:
I already change license content, tks

Reply:
any question?